turingschool/savile
# Text utils

#### What does this PR do?

This PR adds some utils for **text** and adds them to sub-nav of Utils nav bar! The four categories are:

- font-weight
- text-align
- text-decoration
- text-transform

#### Where should the reviewer start?

- `utils.html` - what do you think about the order of all these sub-nav items?
- `utils/text.scss` - I made a separate docstring mixin to be used for text-transform so it's a bit more clear what those do (I think most folks would know, but just want to be as clear as possible!). I kept the each loop for weights since there are 5 and who knows, maybe one day we add more?
- ~~`utils-table.html` template - I added some conditional logic to make sure the p tags are visible when we want, but not when we don't want. I'm not sure if that is the best way to go about it?~~
-  Is it cool that these all live in same file? / are the comment labeling each section ok/helpful?
- Would you like me to add any categories of classes?

#### How should this be manually tested?

If you build the css docs and run, you'll see them on doc page and in the main.css file that is created!

#### Any background context you want to provide?

[I mainly referenced Bootstrap](https://getbootstrap.com/docs/5.0/utilities/text/) as a guide for this, did not go in as much depth as them, but wanted to see what you thought of this as a start.

#### What are the relevant tickets?

https://3.basecamp.com/3494409/buckets/19192671/todos/3354455034

#### Screenshots (if appropriate)

<img width="1552" alt="Screen Shot 2021-04-26 at 11 12 30 AM" src="https://user-images.githubusercontent.com/25447342/116123516-5068c400-a680-11eb-8b3d-f4472aae0a52.png">
<img width="1552" alt="Screen Shot 2021-04-26 at 11 12 17 AM" src="https://user-images.githubusercontent.com/25447342/116123527-5363b480-a680-11eb-9593-0cff2fe031ba.png">
<img width="1552" alt="Screen Shot 2021-05-11 at 10 16 42 AM" src="https://user-images.githubusercontent.com/25447342/117849910-0536df80-b242-11eb-8193-615555dbfb45.png">

#### Any other deploy steps?
